operations/network/management_network/bican_enable.md

```diff
@@ -20,33 +21,70 @@
     - [Validate NCNs](#validate-ncns)
     - [Validate the API gateways](#validate-the-api-gateways)
 
+<a name="configuration-tasks"></a>
+
 ## Configuration Tasks
 
+<a name="configure-sls"></a>
+
 ### Configure SLS
 
-To enable the Customer High Speed Network (CHN) the `SystemDefaultRoute` attribute in the System Layout Service (SLS) `BICAN` network needs to be set to the desired network.
+Configuration for the default route of a `BICAN` enabled system is contained in the System Layout Service (SLS) `BICAN` data structure in the `SystemDefaultRoute` attribute value.
+This structure was created and it's value set during] [Prerequisites Stage 0.2](../../upgrade/1.2/../../../upgrade/1.2/Stage_0_Prerequisites.md#stage-02---update-sls) and can be either `CAN` or `CHN`.
+
+For CSM 1.2 the recommended value for the `BICAN` `SystemDefaultRoute` is `CAN`.
+This allows continued use of UAN, UAI and API resources over the `CAN` and allows a fully supported transition to `CHN` in a later CSM release.
+To update SLS with `CAN` as the `SystemDefaultRoute`:
+
+```bash
+/usr/share/doc/csm/scripts/operations/bifurcated_can/bican_route.py --route CAN
+Setting SystemDefaultRoute to CAN
+```
 
-Run the following command to update SLS with `CHN` as the `SystemDefaultRoute`
+**Preview:** High Speed Network access by users to UAN, UAI and API resources is the `CHN` option.
+This is available during the CSM 1.2 release for those who wish to forge ahead of release-supported features.
+To set and use the `CHN` in SLS update the `SystemDefaultRoute` with:
 
 ```bash
-/usr/share/doc/csm/scripts/operations/bifurcated_can/bican_route.py --route CHN
+ncn# /usr/share/doc/csm/scripts/operations/bifurcated_can/bican_route.py --route CHN
```

Comment:
Ensure command prompts are following the new convention:

https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-conventions

Reply:
> Ensure command prompts are following the new convention:
> 
> https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-conventions

For main branch, we do need to include the command prompt somewhere to indicate where the command is to be run, but it doesn't have to be inside the fenced block itself. This was the change that allowed easier copy and paste. I do think this should probably be more explicitly clarified in the conventions page, though.